app/views/facility_journals/new.html.haml

```diff
@@ -18,6 +18,8 @@
 
 - if @journal_creation_reminder
   = render "journal_creation_modal", reminder_message: @journal_creation_reminder.message
+= render "journal_90day_modal"
+
 = render "shared/transactions/headers"
```

Comment:
We should double check where else this partial is used

Reply:
@giladshanan looks like `shared/transactions/headers` is used 19 times.